src/workflows/foraging.bonsai

```diff
@@ -8066,7 +8130,7 @@
         <Property Name="FileName" DisplayName="SettingsPath" />
       </Expression>
       <Expression xsi:type="io:CsvReader">
-        <io:FileName>C:\Users\svc_aind_ephys\Documents\ForagingSettings\Settings_box1.csv</io:FileName>
+        <io:FileName>C:\Users\xinxin.yin\Documents\ForagingSettings\Settings_box1.csv</io:FileName>
```

Comment:
Is the change of file directory intended?

Reply:
No, this path is from the settings.

Reply:
Does the account change affect access to the file of other users?

Reply:
No, it will not affect access to the file of the other users. The file doesn't need to be changed. I ran it on my own computer and it was changed automatically when I made other changes.

Reply:
The settings file is stored in `C:\Users\user_name\Documents\ForagingSettings\`. We only use one user to train mice in behavior/ephys computer.